src/app/guards/auth-guard.service.ts

```diff
@@ -77,4 +80,8 @@ export class AuthGuard implements CanActivate {
   private checkPlaneLogRoute(): boolean {
     return this.authService.memberHasAuthorization('PASSIVE') ? false : true;
   }
+
+  private checkFinanceRoute(): boolean {
+    return this.authService.memberHasAuthorization('VV') ? true : true;
```

Comment:
Sorry für die verwirrung. DIe Abkürzungen die vom backend in der Apispec angegeben wurden sind nicht die korrekten string die zu erwarten sind. VV -> VORSTANDSVORSITZENDER